src/components/DownloadHeader/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -30,7 +30,7 @@ exports[`DownloadHeader component renders correctly 1`] = `
     <div
       class="npm"
     >
-      (includes npm 6.14.8)
+      (includes npm)
```

Comment:
As @pkgjs/nv doesn't support getting NPM version, I for now, added an issue there.